### PR TITLE
added comments to the elevator class

### PR DESCRIPTION
**Pull request by TrevorBivi**
Adds comments to me code.
Also chances `scheduler =` to `this.scheduler =` like the line above it storing floorAmount to be more clear

### CODE REVIEW

File: ElevatorSim/src/elevatorsim/elevator/Elevator.java [hunk 1]
```diff
@@ -5,28 +5,48 @@
...
+	/**
+	 * Receives message requests from the schedule and then
+	 * sends the messages back to the scheduler but
+	 * targeting floors instead of elevators
+	 * 
+	 * @param message The MessageRequest that should be redirected to floors
+	 */
 	@Override
 	public void recieve(MessageRequest message) {
 		// TODO Auto-generated method stub
```
Comment (rahul-anilkumar) [inline ElevatorSim/src/elevatorsim/elevator/Elevator.java]: Remove this To-Do

File: ElevatorSim/src/elevatorsim/elevator/Elevator.java [hunk 2]
```diff
@@ -5,28 +5,48 @@
...
+	 * The thread just sleeps to allow other threads to run
+	 */
 	public void run() {
 		while (true) {
 			try {
-				//get message
 				Thread.sleep(0);
-				
 			} catch (Exception e) {
 				// TODO Auto-generated catch block
```
Comment (rahul-anilkumar) [inline ElevatorSim/src/elevatorsim/elevator/Elevator.java]: Please remove this one too

File: ElevatorSim/src/elevatorsim/elevator/Elevator.java [hunk 3]
```diff
@@ -5,28 +5,48 @@
 import elevatorsim.enums.MessageDestination;
 import elevatorsim.scheduler.Scheduler;
 
+/**
+ * The elevator class
+ * Currently this is just used to receive messages from the scheduler
+ * and redirect them to the floors
```
Comment (rahul-anilkumar) [inline ElevatorSim/src/elevatorsim/elevator/Elevator.java:11]: Add author? Also should we just put everyones names as the authors for all classes?

**Comment by DwangDistracted** [comment]
I merged the dwang/iteration1Integration branch already. You might need to change the target to master.

File: ElevatorSim/src/elevatorsim/elevator/Elevator.java (hunk 2 above)
Comment (TrevorBivi) [reply ElevatorSim/src/elevatorsim/elevator/Elevator.java]: removed

File: ElevatorSim/src/elevatorsim/elevator/Elevator.java (hunk 1 above)
Comment (TrevorBivi) [reply ElevatorSim/src/elevatorsim/elevator/Elevator.java]: removed

File: ElevatorSim/src/elevatorsim/elevator/Elevator.java (hunk 3 above)
Comment (TrevorBivi) [reply ElevatorSim/src/elevatorsim/elevator/Elevator.java:11]: added author. I put full name then student number in brackets.

**Comment by TrevorBivi** [comment]
> 
> 
> I merged the dwang/iteration1Integration branch already. You might need to change the target to master.

I have now done this

**Review by rahul-anilkumar — APPROVED** [review]

**Pull request merged**